pkgs/applications/misc/kjv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "kjv";
+  version = "unstable-2018-12-25";
+
+  src = fetchFromGitHub {
+    owner = "bontibon";
+    repo = pname;
+    rev = "fda81a610e4be0e7c5cf242de655868762dda1d4";
+    sha256 = "1favfcjvd3pzz1ywwv3pbbxdg7v37s8vplgsz8ag016xqf5ykqqf";
+  };
+
+  add-apocrypha = fetchpatch {
+    url = "https://github.com/LukeSmithxyz/kjv/commit/b92b7622285d10464f9274f11e740bef90705bbc.patch";
+    sha256 = "0n4sj8p9m10fcair4msc129jxkkx5whqzhjbr5k4lfgp6nb1zk8k";
+  };
+
+  patches = [ add-apocrypha ];
+
+  makeFlags = [
+    "PREFIX=${placeholder ''out''}"
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
We mean't to fetch this commit also https://github.com/LukeSmithxyz/kjv/commit/f4ad73539eb73f1890f4b791d8d38dd95900a4a4

Then you won't need this.

Reply:
Fetching the Makefile-tweaking patch doesn't actually strike me as being desirable here, though.  The patch that I've fetched is the only interesting addition that the fork makes; using my own `installPhase` here actually seems a bit cleaner.